python/setup.py

```diff
@@ -272,6 +272,8 @@ def get_packages(self):
             "fastapi",
             "aiorwlock",
             "watchfiles",
+            "numpy >= 1.16; python_version < '3.9'",
```

Comment:
I think `>= 1.16` should be unnecessary given your comment above saying that `< 1.16` was only supported on Python 3.7 and below

Reply:
I left the pin since this particular one could make a difference for Python 3.8, but I think you are right, we can simplify this. Let me do that :)